### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by reinkrul**
Fixes build error

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,7 +4,7 @@ go 1.19
 
 require (
 	github.com/avast/retry-go/v4 v4.3.1
-	github.com/deepmap/oapi-codegen v1.11.0
+	github.com/deepmap/oapi-codegen v1.12.2
```
Comment (gerardsn) [inline go.mod:7]: probably v1.12.3?

**Review by gerardsn — APPROVED** [review]
This increases the total number of dependencies in go.mod + go.sum by >35%. Does `go mod tidy` remove some old deps perhaps?

**Review by gerardsn — CHANGES_REQUESTED** [review]
Should regenerate oapi when updating the dependency

**Comment by reinkrul** [comment]
Reverted

**Pull request merged**